.github/workflows/ci.yml

```diff
@@ -85,9 +88,8 @@ jobs:
         registry: ghcr.io
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
-    - name: Build
-      shell: bash
-      run: make DOCKER_ORG="ghcr.io/bufbuild" DOCKER_BUILD_EXTRA_ARGS="--push"
```

Comment:
This was one thing I wanted to fix from before - it was pushing the images to the registry prior to testing. I want the tests to pass before we push anything.